src/input/InputPlugin.js

```diff
@@ -838,7 +838,43 @@ var InputPlugin = new Class({
      */
     disable: function (gameObject)
     {
-        gameObject.input.enabled = false;
+        var input = gameObject.input;
+
+        if (!input)
+        {
+            return this;
+        }
+
+        input.enabled = false;
+
+        // Clear from _temp, _drag and _over
+        var index = this._temp.indexOf(gameObject);
+
+        if (index > -1)
+        {
+            this._temp.splice(index, 1);
+        }
+
+        for (var i = 0; i < 10; i++)
```

Comment:
Maybe change to `for (var i = 0; i < this.manager.pointersTotal; i++)` for possible optimization?